### ISSUE TITLE
Add element on page that will contain the sought var name

### ISSUE THREAD

**Issue by plocket** [plocket/docassemble-ALAutomatedTestingTests#119]
The one currently in da is going to go away. I'm just going to merge this one straight in because we need it. We may still have trouble with some pages, like ones that were triggered with `mandatory: True`.

**Comment by plocket** [comment]
I take that back - `mandatory: True` pages seem to have a sought var value of 'None' when we add the element in ourselves. That should do just fine I think because of the way we don't require sought vars to be defined.

There are probably tricky situations where it could still come up, like on signature pages that are mandatory or something. Someone make an issue to test that someday.

**Comment by plocket** [comment]
This currently doesn't work on signature screens, so I have to add it to another part of the screen that I'm not sure will work on mobile phones. I don't know of another way to do this right now. That is, I could do it for our tests, but I believe other interviews can't do this easily because they're already using those screen parts and we can't help them because that content is not composable/additive. We can't access the value we need from javascript files that we could include because they can only get the variables that are on the page, but they need to run at daPageLoad and happen before those values get there, I think. I dunno. It didn't work. Not even with `get_interview_variables`. We could do it with some sort of polling function, but that's a bit nuts.

Things we tried with the js:
- `value( 'sought_var' )` with that defined in the python.
- `get_interview_variables`. Don't remember why that didn't work even though I just did it. The sought var code block was `initial: True`, but maybe the page loads before these vars are calculated? I don't remember. Also, a note says that it's slow. Just fyi.
